include/ignition/physics/BoxShape.hh

```diff
@@ -80,6 +80,9 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature rigidly attachs a BoxShape to the specified
+    /// Link.
```

Comment:
Maybe we can elaborate on "rigidly attach a BoxShape" here, something like "This feature constructs a new box shape, and attaches the pose of the box shape relative to the link frame."